app/controllers/project.py

```diff
@@ -13,6 +13,32 @@
 from flask_restful import Resource, request
 from kubernetes import client
 from flask_jwt_extended import jwt_required, get_jwt_identity, get_jwt_claims
+from app.helpers.database_service import MysqlDbService, PostgresqlDbService
+import os
+
+# database flavours
+database_flavours = [
```

Comment:
@MutegekiHenry , to avoid redundancy and having to make changes of the same thing in multiple areas, why don't you import these flavours from the `app.controllers.app.py` file.

Reply:
yeah sure that saves us on redundancy

Reply:
That puts you at risk of falling into cyclic dependencies, if something needs to be re used then define it in a central place, maybe a helper and then every other component that needs to use it imports it from there.

Reply:
Yeah btw this is true

Reply:
Oh just seen this after I made the initial suggested changes. Let me move it to the helpers then